code/modules/mob/living/living.dm

```diff
@@ -88,7 +88,7 @@
 		return
 	if(buckled || now_pushing)
 		return
-	if((confused || is_blind()) && stat == CONSCIOUS && (mobility_flags & MOBILITY_STAND) && m_intent == "run" && (!ismovable(A) || is_blocked_turf(A)) && !HAS_MOB_PROPERTY(src, PROP_CANTBUMPSLAM))  // ported from VORE, sue me
```

Comment:
this is entirely the fault of the previous isblind changes. just make it check if you have the temporary kind of blindness (eyeblind) rather than eyeblind OR trait blindness. consider it a fix